src/test/java/org/betonquest/betonquest/modules/logger/format/ChatFormatterTest.java

```diff
@@ -46,6 +47,7 @@ public ChatFormatterTest() {
         when(pluginExtension.getName()).thenReturn("Extension");
     }
 
+    @Disabled
```

Comment:
You disabled a buch of tests here

Reply:
I did this because I was not certain that you wanted the logger changes as well as the heads support. They might need to be two separate PRs instead. If we're ok with these changes, I can update the tests for the modified logging.